lib/app/preflight_check.rb

```diff
@@ -122,6 +126,8 @@ def fail_check!(message)
     end
   end
 
+  AUTHORIZATION_ERROR = [Elastic::Transport::Transport::Errors::Unauthorized]
```

Comment:
I find it slightly confusing - ERROR is in singular, but contains array and is treated as array. I propose to either make it plural or remove assign just a class, not wrapped in the array.